server/src/main/java/org/elasticsearch/common/logging/ECSJsonLayout.java

```diff
@@ -64,13 +64,12 @@ public EcsLayout build() {
         }
 
         private KeyValuePair[] additionalFields() {
-            return new KeyValuePair[]{
-                new KeyValuePair("type",type),
-                new KeyValuePair("cluster.uuid","%cluster_id"),
-                new KeyValuePair("node.id","%node_id"),
-                new KeyValuePair("node.name","%ESnode_name"),
-                new KeyValuePair("cluster.name","${sys:es.logs.cluster_name}"),
-            };
+            return new KeyValuePair[] {
+                new KeyValuePair("event.dataset", type),
+                new KeyValuePair("elasticsearch.cluster.uuid", "%cluster_id"),
```

Comment:
elasticsearch.* are our own defined in ECS fields? 
or they are just a custom fields which are not defined there?

Reply:
Ry told me that the ECS team recommends prefixing Elasticsearch-specific things like `cluster` and `node` with `elasticsearch.`

Reply:
These fields are not part of ECS but some of them exist in the Filebeat and Metricbeat module: https://www.elastic.co/guide/en/beats/metricbeat/master/metricbeat-module-elasticsearch.html It might be good to align there.